### ISSUE TITLE
Dependencies to Mirantis repo

### ISSUE THREAD

**Issue by gdubcs** [Open-Lens/lens#13]
**Remove dependencies to Mirantis' lensapp repo**
- during the build, **lens-k8s-proxy** is pulled from "https://github.com/Open-Lens/app-k8s-proxy/releases/download/v${args.version}/lens-k8s-proxy-${args.platform}-${args.downloadArch}" (from "packages/ensure-binaries/src/index.ts") which is non-existent
-  other paths seem to point to the forked project ("https://github.com/lensapp/artifact-hub-repositories/releases/download/latest/repositories.json")

**Suggestion**
Either fork those repo or create a Open-Lens/deps repo (or something similar)

**Comment by gdubcs** [comment]
Other dependencies found:
- github.com/Open-Lens/app-extensions
- github.com/Open-Lens/app-extension-samples
- github.com/lensapp/artifact-hub-repositories

**Comment by fbaldo31** [comment]
For k8s lens proxy I suggest to continue with the original : no need to use something different at this point.
For extentions projects I will publish the repositories

**Comment by gdubcs** [comment]
I don't think this is the way to go, we don't when Mirantis is going to pull the plug on that repo as well.
If they decide to remove it, it will break our builds.
The quick way would be to just host the binary on a "Open-Lens/dependencies" if you really don't want the lens-k8s-proxy repo

**Comment by fbaldo31** [comment]
Yes we can fork the repo but my point is why to maintain it while it is maintained by the original team ?

**Comment by fbaldo31** [comment]
Here is the [fork](https://github.com/Open-Lens/lens-k8s-proxyl)

**Comment by gdubcs** [comment]
I agree Fred, I'm not saying we should maintain it (I don't know go-lang anyway), but merely host the binaries so that our OpenLens build doesn't depend on their repo with hard-coded values.

**Comment by fbaldo31** [comment]
I think we are quite good on this topic but there are still related issues like broken links in markdown files.
@gdubcs do you see others tasks to do on it ?

**Comment by gdubcs** [comment]
I'm afraid so. A project-wide search of "lensapp" in my IDE raised "586 matches across 35 files".
Mostly markdown files though
